src/components/pages/Story1/index.tsx

```diff
@@ -1,33 +1,52 @@
 import React from 'react';
 import { useHistory } from 'react-router';
+import Wireframe2 from '../../../assets/neutral_wireframe.png';
```

Comment:
this wireframe should have a better name.. Scene1?

Reply:
Fixed!